corehq/apps/userreports/tests/test_view.py

```diff
@@ -133,6 +133,9 @@ def _delete_everything(cls):
     @classmethod
     def tearDownClass(cls):
         cls._delete_everything()
+        # todo: understand why this is necessary. the view call uses the session and the
+        # signal doesn't fire to kill it.
+        Session.remove()
```

Comment:
the `testrunner` disposing and session closing seems to work for everything except this. i was wondering if the test client somehow runs in a separate thread though i don't see any evidence of that [in the docs](https://docs.djangoproject.com/en/1.8/topics/testing/tools/#the-test-client)